grakn-engine/src/main/java/ai/grakn/engine/tasks/manager/redisqueue/RedisInflightTaskConsumer.java

```diff
@@ -47,18 +50,33 @@
     private static final ObjectMapper objectMapper = new ObjectMapper();
 
     static final int ITERATIONS = 10;
+    private final Meter exceptions;
+    private final Meter processed;
+    private final Meter dead;
+    private final Meter failedMove;
+    private final Duration destroyInterval;
+    private final Meter failedDestroy;
+    private final Meter destroyable;
 
     private Pool<Jedis> jedisPool;
     private Duration processInterval;
     private Config config;
     private String queueName;
 
     public RedisInflightTaskConsumer(Pool<Jedis> jedisPool, Duration processInterval,
-            Config config, String queueName) {
+            Config config, String queueName, MetricRegistry metricRegistry) {
         this.jedisPool = jedisPool;
         this.processInterval = processInterval;
+        // After a certain time we get rid of it
+        this.destroyInterval = processInterval.multipliedBy(10);
         this.config = config;
         this.queueName = queueName;
+        this.exceptions = metricRegistry.meter(name(RedisInflightTaskConsumer.class, "exceptions"));
```

Comment:
How much overhead does the `metricRegistry` add? I ask because to my eye it looks like it is tracking a lot.